libethash/CMakeLists.txt

```diff
@@ -29,13 +29,7 @@ if (NOT CRYPTOPP_FOUND)
 	find_package(CryptoPP 5.6.2)
 endif()
 
-if (CRYPTOPP_FOUND)
```

Comment:
Great!

See also https://github.com/ethereum/cpp-ethereum/issues/3088.   If that is something which you would like to look at, that would be fantastic.